CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## [1.1.0] - TBD
 
+### Added
+- Option to configure s3 storage class for cost optimization
```

Comment:
Could you please add `.` at the end?

Reply:
and while we're being pedantic change s3 to S3 :)

Reply:
This should also go under the "Added" section below so there is only one.

Reply:
See https://keepachangelog.com/en/1.0.0/ if you haven't worked with this file type before.

Reply:
Sure, making those changes